pkg/updater/resultstore/resultstore.go

```diff
@@ -363,10 +363,21 @@ func (t *testResult) properties() []*resultstorepb.Property {
 
 // processGroup will convert grouped invocations into columns
 func processGroup(tg *configpb.TestGroup, group *invocationGroup) *updater.InflatedColumn {
-	if group == nil || group.Invocations == nil {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
i would leave the concurrency for later - let's make that code is working correctly in synchronous manner first. So for now, no need for `contexts` and `channels`

Reply:
done.